### ISSUE TITLE
Type Definition Error

### ISSUE THREAD

**Issue by Borvik** [jwagner/smartcrop.js#110]
Getting this in our builds:
```
>>> project_name
rm -rf dist/* && tsc
Error: ../../common/temp/node_modules/.registry.npmjs.org/smartcrop/2.0.4/node_modules/smartcrop/index.d.ts(25,1): error TS1046: Top-level declarations in .d.ts files must start with either a 'declare' or 'export' modifier.
```

Which seems to be coming from this:
```
const smartcrop: {
  crop(image: CanvasImageSource, options: CropOptions): Promise<CropResult>;
};
export default smartcrop;
```

I'm not a `.d.ts` expert - but looking at other examples (https://github.com/stephenmathieson/node-tlds/issues/11) this may be better done this way (based on their actual index.d.ts, not the suggestions, but I'm not sure and this is untested):

```
declare const smartcrop: {
  crop(image: CanvasImageSource, options: CropOptions): Promise<CropResult>;
};
export = smartcrop;
```

**Comment by jwagner** [comment]
Hi @Borvik ,

Sorry for breaking your build. I'll try to figure out how to fix this as soon as I can.

**Comment by jwagner** [comment]
@Borvik the issue should be fixed in `2.0.5`. A confirmation would be appreciated.

Again, sorry for the hassle.

**Comment by Borvik** [comment]
Thanks @jwagner.  I will test when able and report back - we actually had a different emergency come up that we are still working through.

We actually use `smartcrop-sharp` and hard-coded our dependency to `"smartcrop-sharp": "2.0.3"` to avoid the issue for now.
